canal/handler.go

```diff
@@ -11,6 +11,7 @@ type EventHandler interface {
 	OnRow(e *RowsEvent) error
 	OnXID(nextPos mysql.Position) error
 	OnGTID(gtid mysql.GTIDSet) error
+	OnPosSynced(pos mysql.Position, force bool) error // Use your own way to sync position,When force is true,Immediate sync pos.
```

Comment:
can you move the comment above the function?

Btw, we should add blank after the comma, like
```
Use your own way to sync position. When force is true, sync position immediately.
```